tests/integration/api_v3/http/tokenIDs.test.js

```diff
@@ -18,22 +18,58 @@ const { api } = require('../../../helpers/api');
 
 const {
 	goodRequestSchema,
+	badRequestSchema,
 } = require('../../../schemas/httpGenerics.schema');
 
 const {
-	tokenIDsSchema,
-	tokenIDsMetaSchema,
+	goodResponseSchemaFortokenIDs,
 } = require('../../../schemas/api_v3/tokenIDs.schema');
 
 const baseUrl = config.SERVICE_ENDPOINT;
 const baseUrlV3 = `${baseUrl}/api/v3`;
 const endpoint = `${baseUrlV3}/token/ids`;
 
 describe('Token IDs API', () => {
-	it('retrieves token info when call with address and tokenID-> ok', async () => {
-		const response = await api.get(`${endpoint}`);
+	it('Should retrieves token ids when called without any params', async () => {
+		const response = await api.get(endpoint);
 		expect(response).toMap(goodRequestSchema);
-		expect(response.data).toMap(tokenIDsSchema);
-		expect(response.meta).toMap(tokenIDsMetaSchema);
+		expect(response).toMap(goodResponseSchemaFortokenIDs);
+		expect(response.data.tokenIDs.length).toBeLessThanOrEqual(10);
+	});
+
+	it('Should retrieves token ids when called with offset=1', async () => {
+		const response = await api.get(`${endpoint}?offset=1`);
+		expect(response).toMap(goodRequestSchema);
+		expect(response).toMap(goodResponseSchemaFortokenIDs);
+		expect(response.data.tokenIDs.length).toBeLessThanOrEqual(10);
+	});
+
+	it('Should retrieves token ids when called with limit=5', async () => {
+		const response = await api.get(`${endpoint}?limit=5`);
+		expect(response).toMap(goodRequestSchema);
+		expect(response).toMap(goodResponseSchemaFortokenIDs);
+		expect(response.data.tokenIDs.length).toBeLessThanOrEqual(5);
+	});
+
+	it('Should retrieves token ids when called with offset=1 and limit=5', async () => {
+		const response = await api.get(`${endpoint}?offset=1&limit=5`);
+		expect(response).toMap(goodRequestSchema);
+		expect(response).toMap(goodResponseSchemaFortokenIDs);
+		expect(response.data.tokenIDs.length).toBeLessThanOrEqual(5);
+	});
+
+	it('Invalid limit -> bad request', async () => {
```

Comment:
We can refactor other tests like below

```suggestion
	it('should return bad request when called with Invalid limit', async () => {
```